clients/python/girder_client/__init__.py

```diff
@@ -323,6 +323,12 @@ def authenticate(self, username=None, password=None, interactive=False, apiKey=N
 
             self.setToken(resp['authToken']['token'])
 
+        if not 'user' in resp.keys():
```

Comment:
Let's remove this block, if we got here we can assume that the response contains a `user` field.